python/flask/unsanitized_input.yaml

```diff
@@ -0,0 +1,39 @@
+rules:
+  - id: response_contains_unsanitized_input
+    patterns:
+      - pattern-either:
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              return flask.make_response("...".format($X))
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              $R = flask.make_response("...".format($X))
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              return flask.make_response(f"...{$X}...")
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              $R = flask.make_response(f"...{$X}...")
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              return flask.make_response(f"...{$X}")
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              $R = flask.make_response(f"...{$X}")
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              return flask.make_response(f"{$X}...")
+          - pattern: |
+              $X = request.args.get(...)
+              ...
+              $R = flask.make_response(f"{$X}...")
+    message: Flask response reflects unsanitized user input.
```

Comment:
Need to add more verbose messaging. How about we give more details, similar to https://github.com/returntocorp/sgrep-rules/issues/210
```
 Flask response reflects unsanitized user input. This introduces possible [XSS attack](https://www.owasp.org/index.php/Cross-site_Scripting_(XSS) ) as user can supply input like "<script>some_code</script>" and execute arbitrary code in user's browser.
```